### PR TITLE
Rewording and remove outdated statements, include license history

### PR DESCRIPTION
**Pull request by CoMPaTech**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,18 +50,22 @@ Module providing interfacing with the Plugwise devices:
...
+  - Networked Plugwise Anna module with custom_module - `haanna/anna-ha` via https://github.com/laetificat (Kevin)
+  - USB-based stick module with custom_module - `plugwise-stick/plugwise` by @brefra (Frank)
+  - USB-plugwise module - `plugwise` by https://github.com/cyberjunky/python-plugwise (Ron) originally by https://github.com/aequitas/python-plugwise (Johan) (with reference only in license to Sven)
 
 # Thanks
 
-On behalf of @CoMPaTech and @bouwew (as well as @riemers) we'd like to thank @TANE from [HAshop](https://hashop.nl) for his support and development devices
-
-Also thanks to our numerous testers willing to take the plunge and supply us with actual data from their devices to test and develop from.
+On behalf all of us, big thanks to Plugwise and community members @riemers and @tane from [HAshop](https://hashop.nl)for their support and obviously all our users and testers who dealt our typos and challenges. Disclaimer, while we are communicating with Plugwise and they expressed their gratitude through their newsletter, we are not part of Plugwise as a company. We are just a bunch of guys anxious to get our (and your) Plugwise products working with Home Assistant.
```
Comment (bouwew) [inline README.md:71]: "...dealt **with** our typos.."

Comment (CoMPaTech) [reply README.md]: just before the word typo .... :)

**Review by bouwew — APPROVED** [review]
Nice update!

**Review by brefra — APPROVED** [review]
LGTM

**Pull request merged**